runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineDebugOptions.java

```diff
@@ -191,6 +191,18 @@ public Dataflow create(PipelineOptions options) {
   void setOverrideWindmillBinary(String value);
 
   /**
+   * Custom windmill service endpoint.
+   */
+  @Description("Custom windmill service endpoint.")
```

Comment:
but I think we don't want to use gRPC client for local worker? @dpmills could confirm.